main-service/src/main/java/ru/practicum/main_service/MainServiceApplication.java

```diff
@@ -2,8 +2,11 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Import;
+import ru.practicum.statsclient.StatsClient;
 
 @SpringBootApplication
+@Import({StatsClient.class})
```

Comment:
Для чего этот импорт?

Reply:
У меня без него не запускается проект, разные штуки пробовала, но помогло только это)

```suggestion
***************************
APPLICATION FAILED TO START
***************************

Description:

Parameter 0 of constructor in ru.practicum.main_service.statistic.Statistic required a bean of type 'ru.practicum.statsclient.StatsClient' that could not be found.


Action:

Consider defining a bean of type 'ru.practicum.statsclient.StatsClient' in your configuration.

Disconnected from the target VM, address: '127.0.0.1:49805', transport: 'socket'

Process finished with exit code 1

```